tests/integration/run.py

```diff
@@ -744,17 +744,23 @@ def load_test_cases(
         else:
             # Each --include-tag/--exclude-tag argument adds an additional constraint
             # that a selected test case needs to satisfy, i.e. a selected test case must:
-            # - contains all tags specified with --include-tag
+            # - contains at least on tag specified with --include-tag
```

Comment:
```suggestion
            # - contains at least one tag specified with --include-tag
```

Reply:
Addressed in https://github.com/oracle/macaron/pull/782/commits/874d7577e92969bc663cba56bd98b5fd435e3fa6